esp/services/ws_dfu/CMakeLists.txt

```diff
@@ -27,6 +27,8 @@ project( ws_dfu )
 include(${HPCC_SOURCE_DIR}/esp/scm/smcscm.cmake)
 
 set (    SRCS 
+         ${HPCC_SOURCE_DIR}/common/workunit/package.cpp
```

Comment:
Yes, there was a circular dependency.  This is just my temporary work around while I work on the larger PR.  I'll probably create a new shared library because it's dependency on both dfu and workunit.